woocommerce/woocommerce
# Update footer large Twitter icons to X

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the `Large Footer` and `Large Footer Dark` Twitter icons to X and adds the appropriate URLs.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post
2. Insert the `Large Footer Dark` and the `Large Footer` patterns.
3. Check you don't see any Twitter icons but X ones.
4. Check the URLs of the icons point to the right URL.
5. Save, open the page and check you see the same on the frontend.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Update "Large Footer" and "Large Footer Dark" Twitter icons to X and update the social URLs.
#### Comment 

</details>
